CONTRIBUTING.md

```diff
@@ -36,9 +36,25 @@ In order to run most of the available tests, one would need to run:
 
 `./tools/run_tests/run_tests.py`
 
-If you want to run all the possible tests for any of the languages {c, c++, node, php, python}, do this:
-
-`./tools/run_tests/run_tests.py -l <lang> -c all`
+If you want to run tests for any of the languages {c, c++, node, php, python}, do this:
+
+`./tools/run_tests/run_tests.py -l <lang>`
+
+For list of available commands, refer to this:
```

Comment:
Perhaps we should ask the reader to instead invoke `run_tests.py -h` for themselves to avoid easy-to-forget-potentially-stale documentation in the future (unless we also want a template file for `CONTRIBUTING.md` :laughing:).